Compile error when installing cargo-oudated

On Intel macOS 11.2 using stable rust 1.49.

```
rustc --version
rustc 1.49.0 (e1884a8e3 2020-12-29)
cargo install cargo-outdated
Updating crates.io index
Installing cargo-outdated v0.9.14

...

error[E0283]: type annotations needed
   --> /Users/alexander/.cargo/registry/src/github.com-1ecc6299db9ec823/cargo-0.50.0/src/cargo/util/config/de.rs:530:63
    |
530 |                 seed.deserialize(Tuple2Deserializer(1i32, env.as_ref()))
    |                                                           ----^^^^^^--
    |                                                           |   |
    |                                                           |   cannot infer type for type parameter `T` declared on the trait `AsRef`
    |                                                           this method call resolves to `&T`
    |
    = note: cannot satisfy `std::string::String: AsRef<_>`

error: aborting due to previous error
```

--- a/README.md
+++ b/README.md
@@ -42,12 +42,12 @@ term_size->libc  0.2.18   0.2.29  0.2.29   Normal       cfg(not(target_os = "win
 The latest version of `cargo-outdated` can be installed or updated with `cargo install`:
 
 ```sh
-cargo install cargo-outdated
+cargo install --locked cargo-outdated
 ```
 or
 
 ```sh
-cargo install --git https://github.com/kbknapp/cargo-outdated
+cargo install --locked --git https://github.com/kbknapp/cargo-outdated
 ```
 
 ## Compiling

